### PR TITLE
Various quickfixes

### PR DESCRIPTION
**Pull request by sylvinite**
Aka stuff I should probably bundle together with a big update but will probably forget before it happens.

### CODE REVIEW

File: ngi_pipeline/engines/piper_ngi/database.py [hunk 1]
```diff
@@ -63,6 +63,8 @@ def create_database_populate_schema(location):
     if not os.path.exists( os.path.dirname(location) ):
         try:
             os.makedirs( os.path.dirname(location)) 
+        except OperationalError:
```
Comment (vezzi) [inline ngi_pipeline/engines/piper_ngi/database.py]: are we sure we want to continue in case we are not able to create the directory?

**Comment by vezzi** [comment]
Great!! Will wait for your opinion on my comment on the exception and then merge

File: ngi_pipeline/engines/piper_ngi/database.py (hunk 1 above)
Comment (sylvinite) [inline ngi_pipeline/engines/piper_ngi/database.py]: I had to look it up but try without except crashes the program.
The except is thrown when the directory can't be made (which includes when the directory already exists).
Running without a directory is fine. It's what ngi-pipeline has been capable of doing for quite a while now.

Comment (b97pla) [inline ngi_pipeline/engines/piper_ngi/database.py]: Shouldn't it be an `OSError` though?

Comment (sylvinite) [inline ngi_pipeline/engines/piper_ngi/database.py]: Yea you're probably right. 
OperationalError relates to another problem you can get from ngi-pipeline by trying to access a db you don't own.
Updated.

**Comment by vezzi** [comment]
Perfect

**Pull request merged**